packages/system/src/style.ts

```diff
@@ -224,26 +244,30 @@ export const compose = <TProps = {}>(
   const getStyle = (props: Props<Theme>, sort = true) => {
     const styles = {} as CSSObject
 
+    let generator
     for (const key in props) {
-      const generator = generatorsByProp[key]
+      generator = generatorsByProp[key]
       if (generator) {
         const style = generator.meta.getStyle(props, false)
         merge(styles, style)
       }
     }
 
-    if (!sort) return styles
+    if (!generator || !sort) return styles
```

Comment:
Depending on the order of `for (const key in props)`, is it possible for `generator` to be alternately truthy of falsy at this point?

Reply:
Nice catch, I will change it.